src/test/java/jenkins/plugins/asynchttpclient/AHCTest.java

```diff
@@ -110,24 +111,26 @@ public void failsOnExpiredCertificate() throws Throwable {
         }
         fail("Expired certificate accepted");
     }
-
+    
+    @Issue("JENKINS-60444")
     @Test
     public void acceptGoodCertificate() throws Throwable {
+        String host = "https://8.8.8.8";
```

Comment:
Maybe the variable name could reflect this is a Google DNS address? It might seem it's a local address or something.